server_tests/test/clientTests.js

```diff
@@ -1362,26 +1362,27 @@ describe("RUST API TEST SUITE", function() {
             .then(function onSuccess (res) {
                 Comparison.oscSetOptionsOutput(res);
                 return testClient.bublTimelapse(sessionId);
-            }, shouldNotGetHere)
+            })
             .then(function onSuccess (res) {
                 //Under consistent latency, we can assume the following:
                 //  (1) timelapseRunTime ≈ finalResponseTime - initialResponseTime
                 //  (2) timeElapsed <= timelapseRunTime + pollingPeriod
                 var timeElapsed = res.timeElapsed;
-                Comparison.bublTimelapseOutput(
-                    res,
-                    {
-                        results: {
-                            fileUri: 'bublfile://dcim/100bublc/t0000001.jpg',
-                            _bublFileUris: ['','','']
-                        }
-                    }
-                );
                 if (timeElapsed > maxAcceptableTime) {
-                    done(new Error('operation took too long. timeElapsed : ' + timeElapsed + ' > maxAcceptableTime : ' + maxAcceptableTime));
+                    assert.fail('operation took too long. timeElapsed : ' + timeElapsed + ' > maxAcceptableTime : ' + maxAcceptableTime);
                 } else {
+                    Comparison.bublTimelapseOutput(
+                        res,
+                        {
+                            results: {
+                                fileUri: 'bublfile://dcim/100bublc/t0000001.jpg',
```

Comment:
We can't rely on any specific `fileUri` format, as it's an opaque string subject to change.